### PR TITLE
Updates for spacedock.info

### PR DESCRIPTION
**Pull request by linuxgurugamer**
Updates for mods uploaded to spacedock.info

### CODE REVIEW

File: NetKAN/BetterScienceLabsContinued.netkan [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
     "spec_version"   : "v1.4",
     "identifier"     : "BetterScienceLabsContinued",
-    "$kref"          : "#/ckan/kerbalstuff/903",
+    "$kref"          : "#/ckan/kerbalstuff/45",
```
Comment (Olympic1) [inline NetKAN/BetterScienceLabsContinued.netkan]: change to spacedock

File: NetKAN/KWRocketry-CommunityFixes-interstage.netkan [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "spec_version"  : "v1.2",
     "identifier"    : "KWRocketry-CommunityFixes-interstage",
-    "$kref"         : "#/ckan/kerbalstuff/1036",
+    "$kref"         : "#/ckan/spacedoc/51",
```
Comment (Olympic1) [inline NetKAN/KWRocketry-CommunityFixes-interstage.netkan]: forgot a k

**Comment by linuxgurugamer** [comment]
Thanks

**Comment by linuxgurugamer** [comment]
You have a problem.  The netkan for SpacetuxSA is correct, but when testing the BetterScienceLabsContinued, it continues to try to download the copy from kerbalstuff

**Comment by Dazpoet** [comment]
I seem unable to figure out what the conflicts are so I'm going to say what I've seen people say in these situations and ordinate a Rebase of the PR

**Comment by linuxgurugamer** [comment]
The problem is that it's trying to download spacetuxsa from kerbalstuff.  I have the updated kerbalstuffsa in this commit, so maybe it's one of those strange situations.  Should I redo this, uploading kerbalstufsa first and then the others?

**Comment by Dazpoet** [comment]
No the problem at hand right now is that we cannot merge this due to unresolved conflicts, aka something in this PR doesn't sit well with something else we've changed inbetween the creation of your PR and the current state of the repository. You need to rebase.

**Comment by linuxgurugamer** [comment]
Rebase mean do it all again?

**Comment by Dazpoet** [comment]
@Olympic1 solved this in #3206 :)

Rebase is basically you typing a command in the terminal which makes master/origin/whatever-you-call it match the current upstream and then merge your made changes into it, while prompting you to fix the conflicts which might have arosen.

**Comment by linuxgurugamer** [comment]
Ok, thank you @Olympic1 for taking care of this for me

**Pull request closed without merging**